sdk/batch/azure-compute-batch/src/test/java/com/azure/compute/batch/SharedKeyTests.java

```diff
@@ -41,59 +39,61 @@ public void testPoolCRUD() {
 
             VirtualMachineConfiguration configuration = new VirtualMachineConfiguration(imgRef, nodeAgentSkuId);
 
-            BatchPoolCreateParameters poolCreateParameters = new BatchPoolCreateParameters(sharedKeyPoolId, vmSize);
-            poolCreateParameters.setTargetDedicatedNodes(2)
+            BatchPoolCreateOptions poolCreateOptions = new BatchPoolCreateOptions(sharedKeyPoolId, vmSize);
+            poolCreateOptions.setTargetDedicatedNodes(2)
                                 .setVirtualMachineConfiguration(configuration)
                                 .setTargetNodeCommunicationMode(NodeCommunicationMode.DEFAULT);
 
-            poolClientWithSharedKey.create(poolCreateParameters);
```

Comment:
I should have done a better job before inspecting this but we should assert on the request that the Authorization header contains "SharedKey" to really verify there are no false positives in this test with authenticating shared key.